src/main/java/org/seqdoop/hadoop_bam/BAMInputFormat.java

```diff
@@ -84,6 +88,14 @@
 	public static final String BOUNDED_TRAVERSAL_PROPERTY = "hadoopbam.bam.bounded-traversal";
 
 	/**
+	 * If set to true, enables the use of BAM indices to calculate splits.
+	 * For programmatic use
+	 * {@link #setBAISplitCalculatorEnable(Configuration, boolean)} should be preferred.
+         * By default, this split calculator is disabled in favor of the splitting-bai calculator.
+	 */
```

Comment:
This should be updated to reflect the precedence I suggested in another comment.